python/python/glide/async_commands/cluster_commands.py

```diff
@@ -514,3 +514,42 @@ async def flushall(
             TClusterResponse[TOK],
             await self._execute_command(RequestType.FlushAll, args, route),
         )
+
+    async def lolwut(
+        self,
+        version: Optional[int] = None,
+        parameters: Optional[List[int]] = None,
+        route: Optional[Route] = None,
+    ) -> str:
+        """
+        Displays a piece of generative computer art and the Redis version.
+
+        See https://valkey.io/commands/lolwut for more details.
+
+        Args:
+            version (Optional(int)): Version of computer art to generate.
+            parameters (Optional[str]): Additional set of arguments in order to change the output:
```

Comment:
Same suggestion in other docs
```suggestion
            parameters (Optional[List[int]]): Additional set of arguments in order to change the output:
```